src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -654,6 +654,77 @@
+"""
+
+helps['sql midb ltr-policy show'] = """
+type: command
+short-summary: Show the long term retention policy for a managed database.
+examples:
+  - name: Show long term retention policy for a managed database.
+    text: az sql midb ltr-policy show -g mygroup --mi myinstance -n mymanageddb
+"""
+
+helps['sql midb ltr-backup'] = """
+type: group
+short-summary: Manage SQL Managed Instance database long term retention backups.
+"""
+
+helps['sql midb ltr-backup show'] = """
+type: command
+short-summary: Get a long term retention backup for a managed database.
+examples:
+  - name: Show long term retention backup for a managed database.
+    text: az sql midb ltr-backup show -l southeastasia --mi myinstance -d mymanageddb -n "3214b3fb-fba9-43e7-96a3-09e35ffcb336;132292152080000000"
+"""
+
+helps['sql midb ltr-backup list'] = """
+type: command
+short-summary: List the long term retention backups for a location, instance or database.
+examples:
+  - name: List long term retention backups for a managed database.
+    text: az sql midb ltr-backup list -l southeastasia --mi myinstance -d mymanageddb
+  - name: List long term retention backups for a managed instance (list only the latest LTR backups, which belong to live databases).
+    text: az sql midb ltr-backup list -l southeastasia --mi myinstance --database-state Live --only-latest-per-database True
+  - name: List long term retention backups for a managed instance (with resource group argument).
+    text: az sql midb ltr-backup list -l southeastasia -g mygroup --mi myinstance
+  - name: List long term retention backups for a location (list only the latest LTR backups, which belong to live databases).
+    text: az sql midb ltr-backup list -l southeastasia --database-state Live --only-latest-per-database True
+  - name: List long term retention backups for a location (with resource group argument).
+    text: az sql midb ltr-backup list -l southeastasia -g mygroup
+"""
+
+helps['sql midb ltr-backup delete'] = """
+type: command
+short-summary: Delete a long term retention backup.
+examples:
+  - name: Delete long term retention backup for a managed database.
+    text: az sql midb ltr-backup delete -l southeastasia --mi myinstance -d mymanageddb --name "3214b3fb-fba9-43e7-96a3-09e35ffcb336;132292152080000000"
+"""
+
+helps['sql midb ltr-backup restore'] = """
+type: command
+short-summary: Restore a long term retention backup to a new database.
+examples:
+  - name: Restore a managed database using LTR backup.
+    text: az sql midb ltr-backup restore --dest-database targetmidb --dest-mi myinstance --dest-resource-group mygroup --backup-id "/subscriptions/6caa113c-794c-42f8-ab9d-878d8aa104dc/resourceGroups/mygroup/providers/Microsoft.Sql/locations/southeastasia/longTermRetentionManagedInstances/myinstance/longTermRetentionDatabases/sourcemidb/longTermRetentionManagedInstanceBackups/3214b3fb-fba9-43e7-96a3-09e35ffcb336;132292152080000000"
```

Comment:
Adding `\` in your command will be better format:
![image](https://user-images.githubusercontent.com/49508232/79207622-7fe0d000-7e73-11ea-85c1-ca4c44265296.png)